build.gradle

```diff
@@ -12,10 +12,16 @@ allprojects {
         implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: jacksonVersion
         implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: jacksonVersion
         implementation group: 'org.apache.commons', name: 'commons-lang3', version: apacheCommon
+
+        //Dependency to converter Duration to string and vice versa
         implementation group: 'org.hibernate', name: 'hibernate-core', version: hibernateVersion
 
-        testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junitVersion
-        testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junitVersion
+        implementation group: 'org.springframework.boot', name: 'spring-boot-starter', version: bootVersion
+        implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: bootVersion
+        implementation group: 'org.springframework.boot', name: 'spring-boot-starter-hateoas', version: bootVersion

```

Comment:
Could you please move these dependencies to `web` module?

Reply:
fixed